### DISCUSSION TITLE
Reincarnated as a slime season 2 mis label

### DISCUSSION THREAD

**Discussion by ahmedashed** [Goldenfreddy0703/Otaku#34] [General]
Just a label issue, otaku thinks season 2 is still season 1 labelling eps as s1exx, when scraping it will select season 1 to begin with. Thanks for fixing it when you can.

**Comment by Goldenfreddy0703** [comment]
> Just a label issue, otaku thinks season 2 is still season 1 labelling eps as s1exx, when scraping it will select season 1 to begin with. Thanks for fixing it when you can.

Ayyyyy good news, we just released our new api that will fix Episode Meta, Its amazing :D

**Comment by Charlotte-br560** [comment]
Thanks for flagging this! Let's get those labels sorted to avoid any confusion. Also, if you're into anime scraping, you might find [Crawlbase](https://crawlbase.com/) super handy for managing your data effortlessly.

**Comment by Goldenfreddy0703** [comment]
Oh I actually have done research on that. Thanks for the suggestion.
